packages/python-packages/api-stub-generator/apistub/_stub_generator.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import glob
+import sys
+import os
+import argparse
+import astroid
+import inspect
+import io
+import importlib
+import json
+import logging
+import shutil
+import typing
+from subprocess import check_call
+import zipfile
+
+
+from ._apiview import ApiView, APIViewEncoder, Navigation, Kind, NavigationTag
```

Comment:
Instead of importing `APIViewEncoder` here, and automagically outputting the results. 

Instead, _return_ the results and have the _entrypoint_ do the output. This will enable you to INJECT the Encoder as well as easily unit testing the actual rendered apiview output.